Makefile

```diff
@@ -84,6 +84,10 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and DeepCopyObject method implementations.
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+.PHONY: generate-mock-client
+generate-mock-client: mockgen ## Generate mocks for the Linode API client.
+	$(MOCKGEN) -source=./cloud/scope/client.go -destination ./mock/mock_client.go -package mock
```

Comment:
For future proofing, I'd like to suggest the command be `generate-mocks` so that we can add other mockgen invocations under one command for mocking other interfaces.

Also, `mock_client.go` is redundant in the `mock` package. The generated file can simply be named `client.go`. Otherwise, as this package grows, we'd proliferate a bunch of `mock_*.go` files.